db/schema.rb

```diff
@@ -10,14 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.2].define(version: 2025_10_03_121224) do
+ActiveRecord::Schema[8.0].define(version: 2025_10_15_125032) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "btree_gist"
   enable_extension "citext"
   enable_extension "fuzzystrmatch"
+  enable_extension "pg_catalog.plpgsql"
```

Comment:
This makes me a little bit nervous. Guess if the review app has been published, it is not causing issues with the Azure DB 🤞